modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/paths/APIBuilder.path

```diff
@@ -125,6 +125,11 @@
 	<td>//button[@id='modalDeleteButton']</td>
 	<td></td>
 </tr>
+<tr>
+	<td>NO_API_ENDPOINT_FOUND</td>
+	<td>//span[contains(text(),'No API Endpoint Found')]</td>
+	<td></td>
```

Comment:
you can try to use 'ObjectAdmin#VIEW_ENTRY_NAME_ON_PAGE' to replace it

Reply:
Fixed, Thanks!